java/websvc.jaxws21/nbproject/project.xml

```diff
@@ -52,6 +52,10 @@
             <friend-packages>
                 <friend>org.netbeans.modules.websvc.jaxwsmodel</friend>
                 <friend>org.netbeans.modules.websvc.manager</friend>
+                <package>com.sun.tools.ws</package>
+                <package>com.sun.tools.ws.ant</package>
+                <package>com.sun.tools.ws.api</package>
+                <package>com.sun.tools.ws.api.wsdl</package>
```

Comment:
Are the new exports in this file needed for WSDL generation or what is the motivation? My curiosity was triggered by the  `com.sun.tools.ws.ant` package.